src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -404,4 +412,18 @@ public static Builder builder() {
       public abstract Healthcheck build();
     }
   }
+
+  @AutoValue
+  public abstract static class NetworkingConfig {
```

Comment:
This one we do need. It's part of `ContainerConfig`, which is returned from docker on container inspect.